core-bundle/src/Resources/config/services.yml

```diff
@@ -108,6 +108,13 @@ services:
         #    message: Using the "%alias_id%" service ID has been deprecated and will no longer work in Contao 5.0. Please use "Contao\CoreBundle\Controller\BackendCsvImportController" instead.
         public: true
 
+    Contao\CoreBundle\Controller\ContentElement\MarkdownController:
+        factory: ['Contao\CoreBundle\Controller\ContentElement\MarkdownController', 'createController']
+        arguments:
+            - '@request_stack'
```

Comment:
This will make the controller/service be factored according to the first request that needs it, I don't think that's correct (also see my comment about the factory method).